DQM/SiPixelPhase1Track/plugins/SiPixelPhase1TrackResiduals.cc

```diff
@@ -81,6 +92,13 @@ namespace {
         histo[RESIDUAL_X].fill(it.resXprime, id, &iEvent);
         histo[RESIDUAL_Y].fill(it.resYprime, id, &iEvent);
 
+        if (it.resXprimeErr != 0 && it.resXprimeErr != 0) {
```

Comment:
I can't understand the logic here, isn't this duplicated twice?

Reply:
@arossi83 can you please clarify this, or remove the duplication?

Reply:
Sorry, I just realize where the duplication is, obviously is a typo. I just fixed it!